Project.toml

```diff
@@ -41,8 +41,8 @@ GPUArraysCore = "0.1"
 JSON = "0.21"
 KernelAbstractions = "0.9"
 MuladdMacro = "0.2"
-PointNeighbors = "0.4.2"
-Polyester = "0.7.5"
+PointNeighbors = "0.4.5"
+Polyester = "0.7.15"
```

Comment:
But do we need this version? Or is it running just fine with 0.7.5?

Reply:
There was an error in 0.7.6 that's why we put 0.7.5 here. We can revert to 0.7 if you want?

Reply:
But I don't see any drawback by requiring the newer version since there are no downstream package changes (I checked)

Reply:
Ok so lets progress over the error fix than to 0.7.8?

Reply:
Did we? `"0.7.5"` matches every `0.7.X` version since `0.7.5`, so also `0.7.6`.

Reply:
Than this is wrong anyway?

Reply:
If there was indeed a bug in 0.7.6 and it was fixed in 0.7.8, then I'm fine with setting this to 0.7.8.

Reply:
So 0.7.10 than like trixi.jl? Since there is another error fixed in 0.7.10 that is relevant?

Reply:
Then 0.7.10.